_events/sr2019/competition.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Competition
+date: 2019-04-06 9:00:00
+layout: event
+type: competition
+location: Southampton University Highfield Campus
+---
+
+## Tickets
```

Comment:
Are we actually doing tickets? I know that we didn't do them for SourceBots (and saved quite a lot of work as a result).
We should work out whether we're going to do them before this goes out.